README.md

```diff
@@ -228,6 +251,72 @@ ignore (default: `0`)
 - `-latency-p50`: Expected maximum latency for 50th percentile of requests, 0 to
 ignore (default: `0`)
 
+## Observability & Troubleshooting
+
+### What's happening with the rollout of my service?
+
+To view the current state of the rollout of a particular service, you can see
+the service's traffic configuration in [Cloud
+Run](http://console.cloud.google.com/run).
+
+Under the `Revisions` section, the first thing to notice is that the Release
+Manager has automatically assigned the `stable` tag to your current stable
+revision and the `candidate` tag to your last deployed revision. You can also
+see the percentage of traffic each revision is serving.
+
+#### Annotations
+
+For more detailed information on the rollout process, the service object
+contains some annotations with information about the rollout. In Cloud Run, you
+can see the service object in the `YAML` section.
+
+- `rollout.cloud.run/stableRevision` is the name of the current stable revision
+- `rollout.cloud.run/candidateRevision` is the revision name of the current
+  candidate
+- `rollout.cloud.run/lastFailedCandidateRevision` is the last revision that was
+  considered a candidate but failed to meet the health criteria at some point of
+  its rollout process
+- `rollout.cloud.run/lastRollout` contains the last time a rollout occurred
+  (traffic to the candidate was increased)
+- `rollout.cloud.run/lastHealthReport` contains information on why a rollout or
+  rollback occurred. It shows the results of the health assessment and the
+  actual values for each of the metrics
```

Comment:
this section is way too long also has titles like "Annotations" which is not useful to users. you need to put yourself in user's shoes and read this.

All you're trying to say here is:


```
To check your rollout status:

1. Go to Cloud Console → Cloud Run → click on your service → click Revisions tab and see how your traffic is currently split among revisions.

2. You can also use the annotations added by Release Manager on the Service. To view these, click to the YAML tab of your Service on Cloud Run Console.

     <<example annotation>>

     * explanation of fields

```